index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Gaia Media Controls Demo</title>
+  <meta name="description" content="Demo app for Gaia media controls component">
+  <script src="bower_components/gaia-icons/gaia-icons.js"></script>
+  <script src="bower_components/gaia-component/gaia-component.js"></script>
+  <script src="lib/media-controls-impl.js"></script>
+  <script src="gaia-media-controls.js"></script>
```

Comment:
I don't get why you're including these two raw files instead of the one dist/gaia-media-controls.js file.  I see that gaia-header does its demo this way too, however.  It seems less useful as a demo, though, if it is not how people would actually use the component.